lib/issue_lib/cli.ml

```diff
@@ -20,7 +20,7 @@ let project_dir () =
   | Some x -> x
   | None ->
     failwith (
-      Printf.sprintf "issue directory could not be found. use '%s open' to create issues."
+      Printf.sprintf "issue directory could not be found, use '%s init' to create one."
```

Comment:
use **issue init** to start creating issues. 

How hard would it be to get bold working?